src/site/content/en/fast/choose-the-right-image-format/index.md

```diff
@@ -171,7 +171,7 @@ As a result, the choice of the "right format" for a particular image is a combin
   <td data-th="format"><a href="http://en.wikipedia.org/wiki/WebP">WebP</a></td>
   <td data-th="transparency">Yes</td>
   <td data-th="animation">Yes</td>
-  <td data-th="browser">Chrome, Opera, Android</td>
+  <td data-th="browser">Chrome, Opera, Android, Safari (from version 14)</td>
```

Comment:
It's also supported in Firefox since version 65 (early 2019). Not sure what's the best way to phrase this...

Reply:
maybe we should phrase it at this point as "all modern browsers" and then link that to MDN/Can I Use?

Reply:
I completely destroyed my local branch trying to rebase it so I'll need to start over with it to make the change.

Reply:
ok I've done this.

If I rebase, I will end up with all of the individual commits listed like last time, which was not what was wanted. Is there something I should be doing to stop that happening?

Reply:
I don't think the commit history matters much if we do squash and merge, paging @robdodson for advice :) 

Context: This branch has the outdated version of the /fast page and I suggested rebasing. The lazy-loading section has been added in the meantime and lazysizes post has been moved there. (I suggested rebasing to make sure we don't end up with duplicated lazysizes post because I'm not sure if this would show up as a conflict when you attempt merging.)

Reply:
> I don't think the commit history matters much if we do squash and merge, paging @robdodson for advice :)

Yeah if you're doing a squash and merge from a PR branch then all of the commits will get combined into one.

> (I suggested rebasing to make sure we don't end up with duplicated lazysizes post because I'm not sure if this would show up as a conflict when you attempt merging.)

That's a good question. One option would be to manually do a squash merge to see what happens. I think the commands would be something like this:

```
git checkout -b rachelandrew-image-optimize master
git pull https://github.com/rachelandrew/web.dev.git image-optimize
git merge --squash rachelandrew-image-optimize
```